File: problemas/matematica/1585/1585.lua
n = tonumber(io.read(), 10)
for _ = 1, n do
  entrada = {}
  for numero in io.read():gmatch("%S+") do
    entrada[#entrada + 1] = numero
  end
  x = tonumber(entrada[1], 10)
  y = tonumber(entrada[2], 10)

  print(string.format('%d cm2', (x * y) // 2))
end

File: problemas/matematica/1585/1585.clj
(require '[clojure.string :as str])

(let [n (read-string (read-line))]
  (doseq [_ (range 0 n)]
    (let [linha (read-line)]
      (when linha
        (let [entrada (str/split linha #" ")
              x (read-string (nth entrada 0))
              y (read-string (nth entrada 1))]
          (printf "%d cm2\n" (quot (* x y) 2)))))))
